apps/src/templates/teacherDashboard/SectionTable.jsx

```diff
@@ -52,53 +56,53 @@ class SectionTable extends Component {
       <table style={tableStyle}>
         <thead>
           <tr style={styles.headerRow}>
-            <td style={styles.col}>
+            <td style={{...styles.col, ...styles.headerRowPadding}}>
```

Comment:
We use this so much, let's compose it once earlier in the render method or even in the module scope.
```jsx
const colHeaderStyle = {...styles.col, ...styles.headerRowPadding};
```
Then use _that_ in our JSX
```jsx
<td style={colHeaderStyle}>
```